dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,45 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    @Test
+    public void transformsTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = null;
+        try {
+            outBigDec = (BigDecimal) bdt.transform(nbd);
+        } catch (SQLException e) {
+            fail("InformixBigDecimalTransformer threw SQLException transforming NormalisedBigDecimal input");
+        }
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsInputTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        boolean caught = false;
+        String expectedMsg = "InformixBigDecimalTransformer cannot transform objects of type java.math.BigDecimal";
```

Comment:
> Should we somehow indicate the direction of the transformer in the class name - e.g. `*OutTransformer` or `*OutgoingTransformer` or `<Object>ToJDBCTransformer`...?

I think that the *direction* element can be omitted from the actual classes implementing `TypeTransformer` and instead placed in the `TypeTransformerFactory` object instance name. E.g. `dbfitToJDBCTransformerFactory` (this clearly states the source and target for the collection).

If we go with this scheme then perhaps we should rename `TypeNormaliserFactory` to something like `jdbcToDbFitTransformerFactory` (or `jdbcToDbFitNormaliserFactory`). I do like the `Normaliser` part of the name.

What do you think of this alternative?